vm/rust/src/lib.rs

```diff
@@ -350,6 +351,13 @@ pub extern "C" fn cairoVMExecute(
 
                 let actual_fee = t.transaction_receipt.fee.0.into();
                 let data_gas_consumed = t.transaction_receipt.da_gas.l1_data_gas.into();
+                let execution_steps = t
+                    .transaction_receipt
+                    .resources
+                    .vm_resources
+                    .n_steps
+                    .try_into()
+                    .unwrap_or(u64::MAX);
```

Comment:
why it's u64::MAX if conversion is failed?

Reply:
the only time a conversion actually fails is when coming from a higher range to a lower range.
it fails in the case of overflow.

Reply:
makes sense, thanks!